website/source/docs/templates/user-variables.html.md

```diff
@@ -162,3 +166,24 @@ provisioner only run if the `do_nexpose_scan` variable is non-empty.
   "command": "if [ ! -z \"{{user `do_nexpose_scan`}}\" ]; then python -u trigger_nexpose_scan.py; fi"
 }
 ```
+
+## Using HOME Variable
+
+In order to use `$HOME` variable, you can create a `home` variable in packer:
+
+``` {.javascript}
+"variables" {
```

Comment:
A colon is missing, i.e. `"variables": {` instead of `"variables" {`

Reply:
Hello @030, thanks for commenting, this is a very old pull request and this is not the case anymore: you can find the master version here : https://github.com/hashicorp/packer/blob/master/website/source/docs/templates/user-variables.html.md#using-home-variable

Reply:
@azr Ok thanks. Next time I will check the master branch first.